chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/summarystatistics/StatsCsvFileWriter.java

```diff
@@ -31,66 +30,44 @@
     };
 
     public void writeCsvFile(String fileName, List<Statistics> statsCsvOutput) {
-        FileWriter fileWriter = null;
-        CSVPrinter csvFilePrinter = null;
+        
         CSVFormat csvFileFormat = CSVFormat.DEFAULT.withRecordSeparator(NEW_LINE_SEPARATOR);
         SimpleDateFormat dateFormat = new SimpleDateFormat("E MMM dd yyyy");
 
         try {
-            // initialize FileWriter object
-            fileWriter = new FileWriter(fileName);
-
-            // initialize CSVPrinter object
-            csvFilePrinter = new CSVPrinter(fileWriter, csvFileFormat);
-
-            csvFilePrinter.printRecord(FILE_HEADER);
-
-            // Write a new StatisticsCSVOutput object list to the CSV file
-            for (Statistics stat : statsCsvOutput) {
-                List<String> statRecord = new ArrayList<String>();
-                dateFormat.setTimeZone(TimeZone.getTimeZone(ZoneOffset.UTC));
-                String dateString = dateFormat.format(stat.getDateRange().getEndDate());
-                statRecord.add(dateString);
-                statRecord.add(String.valueOf(stat.getTotalDevelopers()));
-                statRecord.add(String.valueOf(stat.getTotalDevelopersWith2014Listings()));
-                statRecord.add(String.valueOf(stat.getTotalDevelopersWith2015Listings()));
-                statRecord.add(String.valueOf(stat.getTotalCertifiedProducts()));
-                statRecord.add(String.valueOf(stat.getTotalCPsActive2014Listings()));
-                statRecord.add(String.valueOf(stat.getTotalCPsActive2015Listings()));
-                statRecord.add(String.valueOf(stat.getTotalCPsActiveListings()));
-                statRecord.add(String.valueOf(stat.getTotalListings()));
-                statRecord.add(String.valueOf(stat.getTotal2014Listings()));
-                statRecord.add(String.valueOf(stat.getTotal2015Listings()));
-                statRecord.add(String.valueOf(stat.getTotal2011Listings()));
-                statRecord.add(String.valueOf(stat.getTotalSurveillanceActivities()));
-                statRecord.add(String.valueOf(stat.getTotalOpenSurveillanceActivities()));
-                statRecord.add(String.valueOf(stat.getTotalClosedSurveillanceActivities()));
-                statRecord.add(String.valueOf(stat.getTotalNonConformities()));
-                statRecord.add(String.valueOf(stat.getTotalOpenNonconformities()));
-                statRecord.add(String.valueOf(stat.getTotalClosedNonconformities()));
-                csvFilePrinter.printRecord(statRecord);
+            try (FileWriter fileWriter = new FileWriter(fileName);
+                    CSVPrinter csvFilePrinter = new CSVPrinter(fileWriter, csvFileFormat);) {
```

Comment:
Not sure if the last `;` is needed in the list. I don't have it in any of mine, and think it looks odd...

Reply:
Fixed